src/locales/en.json

```diff
@@ -169,7 +169,7 @@
     "risk_tolerance_3_percent_label": "Low Caution Budget",
     "risk_tolerance_3_percent_explanation": "3% chance of COVID per year (600 microCOVIDs per week). If you are healthy and prefer to live a riskier lifestyle AND everyone you live with / share air with is willing to do the same.",
     "risk_tolerance_1_percent_label": "Standard Caution Budget",
```

Comment:
it might make sense to switch this to "Unvaccinated Caution Budget" or something after 2% appears in the calculator.

Reply:
also maybe "Unvaccinated Standard Budget" and "Vaccinated Standard Budget"?

Reply:
Done.